configure.ac

```diff
@@ -60,10 +60,12 @@ AC_PROG_LIBTOOL
 AC_PROG_RANLIB
 AC_HEADER_STDC
 AC_CHECK_HEADERS([infiniband/verbs.h],,[AC_MSG_ERROR([ibverbs header files not found])])
+AC_CHECK_HEADERS([pci/pci.h],,[AC_MSG_ERROR([pciutils header files not found])])
```

Comment:
Please, add check here and below that it is not a FreeBSD machine as we don't support relaxed ordering there. Let's also update README that it is a new dependency on Linux

Reply:
Will move _AC_CHECK_HEADERS_ and _AC_CHECK_LIB_ into this section:
```
 if [test $IS_FREEBSD = no]; then
+       AC_CHECK_HEADERS([pci/pci.h],,[AC_MSG_ERROR([pciutils header files not found])])
+       AC_CHECK_LIB([pci], [pci_init], [LIBPCI=-lpci], AC_MSG_ERROR([libpci not found]))
        CPU_IS_RO_COMPLIANT=yes
        # Actually this is check for being affected by a known issue
        # with Intel CPUs:
```